.github/workflows/merge_to_master.yml

```diff
@@ -0,0 +1,87 @@
+# CI stages to execute against master branch on PR merge
+name: Update Nailgun package to PyPI on merge
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+    PYCURL_SSL_LIBRARY: openssl
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # python 3.6 is still used by IQE integration pipeline
+        python-version: [3.6, 3.8, 3.9]
```

Comment:
let's add 3.10, perhaps dropping 3.8 or 3.9

Reply:
Removed `3.9` and added `3.10` which now checks in even python versions of 6,8 and 10.